### PR TITLE
ISS Form modification

### PR DESCRIPTION
**Pull request by ODORA0**

### CODE REVIEW

**Comment by semujju** [comment]
Rename Nutrition Assessment to Nutrition Status
Screenshot from 2021-07-11 16-06-09

**Comment by ODORA0** [comment]
@semujju Done

**Comment by semujju** [comment]
1. The encounter part added on the health education page has some fields that are not clear, like these below

- PSS Issues Identified
- Depression Status
- Remove the last Comment row
- Put "Is Visit On Schedule" on the first row after encounter date
- Add OVC Assessment
- Add field for specify other PHDP Components
- Add field for specify other linkages and referrals

2. On capture vitals form use sentence case for Nutrition support and NOT upper case
3. Unhide all forms under Lab (location)

File: omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-HealthEducationPage.xml [hunk 1]
```diff
@@ -126,6 +126,19 @@
 							</span>
                     </td>
                 </tr>
+
+                <tr class="enableDisable">
+                    <td>Is Visit On Schedule:</td>
+                    <td id="trigger">
+                        <obs id="msm_final"  conceptId="165134" answerConceptId="160578" answerLabel="Is Visit On Schedule" class="vertical-layout-options"/>
```
Comment (ODORA0) [inline omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-HealthEducationPage.xml:133]: @semujju The right Concept for this field is 90069 but using that will throw a 
`java.lang.RuntimeException: Boolean with style = null not yet implemented (concept = 90069)`

**Pull request merged**